hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSender.java

```diff
@@ -92,7 +102,10 @@ private void sendAsync(Message message, int delayMillis) {
     public void sendMessage(final Message message) {
         rateLimiter.acquire();
         ConsumerLatencyTimer.Context timer = consumerLatencyTimer.time();
-        CompletableFuture<MessageSendingResult> response = async.within(messageSender.send(message), Duration.ofMillis(asyncTimeoutMs + requestTimeoutMs));
+        CompletableFuture<MessageSendingResult> response = async.within(
```

Comment:
\[FindBugs\] INFO: IS: Inconsistent synchronization of pl\.allegro\.tech\.hermes\.consumers\.consumer\.ConsumerMessageSender\.messageSender; locked 50% of time